content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -105,21 +105,25 @@ average and tail latencies. The actual tail latency depends on the traffic patte
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 1.31 ms and 1.58 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+the two proxies add about 0.175 ms and 0.235 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
 for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
 
-{{< image width="90%"
-    link="latency_p90_fortio_with_jitter.svg"
+Note: This testing was performed on the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster).
```

Comment:
do we want to note that different environments provide different performance results?